src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected static final String yesIcon = "[\u2713]";
+    protected static final String noIcon = "[\u2718]";
```

Comment:
Can consider consolidating these into a .class file and import the file when needed